.rubocop_todo.yml

```diff
@@ -0,0 +1,49 @@
+# This configuration was generated by
+# `rubocop --auto-gen-config`
+# on 2016-03-27 01:01:12 +0100 using RuboCop version 0.38.0.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# I don't have a preference.
+Lint/Loop:
+  Enabled: false
+
+# I'd rather use context_dependent, even if I'm not even sure we should enforce a style.
+# SupportedStyles: braces, no_braces, context_dependent
+Style/BracesAroundHashParameters:
+  Enabled: false
+  EnforcedStyle: context_dependent
+
+# Offense count: 39
+Style/Documentation:
+  Enabled: false
+  Exclude:
+    - 'spec/**/*'
+    - 'test/**/*'
+
+# I was a big fan of leading, but after I had some weird compile issues, I reverted
+# to trailing. Any proposal?
+# SupportedStyles: leading, trailing
```

Comment:
I'm neutral. We can start with trailing, and change to leading if we feel it's better. It should be handled automatically by rubocop.